config/initializers/postgresql_timestamp.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.after_initialize do
+  # Use timestampz to create new timestamp columns, so that we get WITH TIME ZONE support
+  ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.datetime_type = :timestamptz
```

Comment:
Rails guide recommends [doing it in an initializer](https://guides.rubyonrails.org/configuring.html#activerecord-connectionadapters-postgresqladapter-datetime-type), but mentions nothing about using a `after_initialize`. Did you run into troubles when doing it outside of the `after_initialize`?

Reply:
I first tried it in application.rb where it failed with a constant lookup error, so I thought it wanted autoloading. But it appears to work fine without the `after_initialize`. I'll remove it